src/app/route/route-edit/route-edit.component.ts

```diff
@@ -95,7 +95,11 @@ export class RouteEditComponent implements OnInit {
     if (this.editMode) {
       this.router.navigate(['../../../yourRoutes/' + this.id], { relativeTo: this.activatedRoute });
     } else {
-      this.router.navigate(['../' + this.id], { relativeTo: this.activatedRoute });
```

Comment:
Something like `this.router.navigate(['../list'], { relativeTo: this.activatedRoute });` wouldn't work?